README.md

```diff
@@ -68,4 +68,96 @@ By default all image uploads are uploaded to a cloud provider of your choice. I
 
 ## Installation
 
-Coming soon...
+### Server
+
+* Clone this repo!
+* Download and install https://getcomposer.org/download/
+
+```
+php composer.phar install
+```
+
+* Follow instructions @ https://laravel.com/docs/5.7/homestead to setup homestead
+* Edit `Homestead.yaml`, map the knot-server folder, change the hostname of the server to knot-server.test
+* In `/etc/hosts` map knot-server.test to the ip listed in Homestead.yaml
+
+```
+vagrant up
+```
+
+* Login to server: `vagrant ssh`
+* Setup database user + db
+
+```
+mysql
```

Comment:
With Homestead, you shouldn't need to create the database user manually. See https://laravel.com/docs/5.7/homestead#connecting-to-databases